Stoovles/quant_self
# 1jest testing

Adds test to knexfile.js (DON'T FORGET TO CHANGE USERNAME)
Adds quant_self_test database

Adds app.spec.js
- test db migrates/seeds from dev 

Tests check for status codes and response.body
- DELETE /foods/:id not passing - maybe because of database relationships?
- DELETE /meals/:meal_id/foods/:id - doesn't return any body so not being tested

Future iterations of tests written for adding an api_key to the body
- will need to have migration for user with api_key of 1234
